scripts/imgtool/main.py

```diff
@@ -441,6 +453,8 @@ def convert(self, value, param, ctx):
               help='send to OUTFILE the payload or payload''s digest instead '
               'of complied image. These data can be used for external image '
               'signing')
+@click.option('--psa-key-ids', multiple=True, type=int, required=False,
```

Comment:
I believe `multiple=True` is not needed if it takes a list of IDs.

Reply:
With multiple =true, Click automatically parses  all cases where 1. No key ids are passed. 2 single key id is passed and 3 multiple key ids are passed